Update WASM Medley description

pamoroso opened the pull request:
This PR updates the [Software](https://interlisp.org/software) page of the project site to describe and link to a running instance of WebAssembly Medley. It addresses [Medley issue #2019](https://github.com/Interlisp/medley/issues/2019).

pamoroso committed:
Update WASM Medley description

Update the description of WebAssembly Medley to link to a running instance and provide some instructions and caveats. Address Medley issue #2019.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

masinter reviewed (changes requested):
I thought we were working on supporting current Medley master to be made available, like online. I'm not sure where the "older version of" came from, but I think it's a mistake. @fghalasz did some experiments with putting Darkstar up on Docker as well.

@ekaltman mentioned some other work on getting WASM code to work with a file system.
This should get sorted out.

pamoroso commented:
The idea behind this change is to point to the current experimental WASM version as an interim step until we replace Medley Online. Possible alternatives are to remove the mention of WASM, or update it to note we're working on a WASM version that will eventually replace Medley Online.

pamoroso committed:
Note WASM limitations, hint alternate approaches

Change the WASM discussion to explain why we don't recommend this approach, hint at alternatives, and encourage help and feedback.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

pamoroso commented:
Based on the feedback of the February 19, 2025 external meeting I changed the WASM discussion to explain why we don't recommend this approach, hint at alternatives, and encourage help and feedback.

pamoroso committed:
Remove mentions of earlier Medley versions

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

pamoroso commented:
I removed the mentions of earlier versions of Medley.

masinter reviewed (approved):
LGTM now
